addons/pos_cash_rounding/views/account_cash_rounding_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="pos_rounding_form_view_inherited" model="ir.ui.view">
+        <field name="name">pos.cash.rounding.form.inherited</field>
+        <field name="model">account.cash.rounding</field>
+        <field name="inherit_id" ref="account.rounding_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_title')]" position="before">
+                <div class="o_notification_alert alert alert-warning" role="alert">
+                  The Point of Sale only support the "add a rounding line" rounding strategy.
```

Comment:
If we add a domain and a constrain, I don't think an extra notification is necessary. By the way, it should be `supports`.